src/reporter.ts

```diff
@@ -94,6 +94,8 @@ export class RPReporter implements Reporter {
       extendTestDescriptionWithLastError: true,
       ...config,
       launchId: process.env.RP_LAUNCH_ID || config.launchId,
+      // TODO: consider the "grep" and "grep-invert" options
+      
```

Comment:
You can use [`onBegin`](https://playwright.dev/docs/test-reporters#custom-reporters:~:text=implements%20Reporter%20%7B-,onBegin,-(config%3A) method to access `grep` and `grepInvert` options. If they are present in `config` parameter, they should be added to the launch attributes in the next format:
- For config option `{ grep: '@fast' }` the launch attribute should be `{ key: 'grep', value: '@fast' }`;
- For config option `{ grepInvert: '@fast' }` the launch attribute should be `{ key: 'grepInvert', value: '@fast' }`.